### DISCUSSION TITLE
Enhanced Plastic tool

### DISCUSSION THREAD

**Discussion by Bamdadmj** [opentoonz/opentoonz#6479] [Ideas]
Hello everybody. I think it would be great if the "create mesh" wizard in the "plastic tool" had an option to be guided by the user. I have to refer to "Pixologic Zbrush" in which the user can guide the "Zremesher" process by drawing guidelines via the "ZremesherGuides" brush. This capability is significantly helpful to obtain a 3D mesh with a clean topology, almost ready to be animated. One of the main advantages of this method is that the resulted mesh could be far less dense and yet be deformed properly. Please let me know if u need more info or pictures to explain what I mean more clearly.
regards
![ZremesherGuide brush screenshot](https://user-images.githubusercontent.com/95492260/144651875-a96d64bb-d677-46a4-9a80-ee3b351de01c.jpg)

**Comment by beeheemooth** [comment]
To make an analogy with ZBrush, the Plastic Mesh in our case is "Dynamesh", where the mesh topology assumes an even distribution of mesh points for good sculpting (in our case bending). Trust me, if you have a perfect mesh topology (as if you were using ZRemesher or manual retopology from ZSpheres), bending will still not be any better, since the bones do not have an adjustable zone of influence on the mesh, the Falloff mode, they cannot inflate mesh or affect the selected mesh vertices. Your request should probably start with developing the functionality of bones. Rough editing of the mesh is also possible in the "Edit Mesh" mode, you can cut the mesh, remove edges or points, e.t.c.
Manual mesh creation will be too time consuming if you decide to create mesh on a sub-xsheet, where each sub-xsheet frame will have its own mesh and many skeletons.

**Comment by Bamdadmj** [comment]
Thanks for your precise reply, Beeheemooth. So, I'll be grateful to see any improvement in functionality of bones. And yes, right now I'm using the "Edit Mesh" options to achieve better bending.  I guess it would be much easier and faster if the zone of influence of bones could be defined.

**Comment by RodneyBaker** [comment]
Tagging this report with 'roadmap' mostly because it states the obvious:  the Plastic Tool needs to be developed/updated.

As we can now have sub-issues we'll try to get one primary report focusing on Plastic Tool and link other related reports to that.

Edit:  As we now have Discussions will convert this for additional investigation.
